.github/workflows/publish.yml

```diff
@@ -14,7 +14,9 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v2
-
+      - name: Get release version
+        id: get_version
+        run: echo ::set-env name=RELEASE_VERSIONS::$(echo "${GITHUB_REF}" | sed -e "s/refs\/tags\///g" | sed -E "s/v?([0-9]+)\.([0-9]+)\.([0-9]+)(-[a-zA-Z]+(\.[0-9]+)?)?/\1.\2.\3\4,\1.\2\4,\1\4/g")
```

Comment:
You can use `${{ github.event.release.tag_name }}` to get the tag instead of parsing the ref.

Reply:
Outdated as your fork now removes the need for any of this